src/System/Components/PageLoadingBar.tsx

```diff
@@ -44,18 +47,19 @@ export const PageLoadingBar: React.FC<PageLoadingBarProps> = ({
         ref={barRef as any}
         loading={loading}
         isComplete={isComplete}
+        scaleX={scaleX}
```

Comment:
So there's only 2 values for this? 0 and 0.3? (Just want to be concerned with generating unnecessary styles)

Reply:
this code is crazy, but all of it has a purpose. 0 -> .3 is the intial ease in, which then extends a bit for slow loads. Then... (next comment)